Gradata/skills/core/session-start/SKILL.md

```
@@ -0,0 +1,52 @@
+5. Read `C:/Users/olive/SpritesWork/brain/brain_prompt.md` (soul.md VOICE mandatories + graduated RULE-level lessons)
+
+## Step 3: Surface Alerts
+
+Only if relevant:
+- Stale files (loop-state, startup-brief >7 days old)
+- Agent job queue directives from hook output (ACTION REQUIRED items)
+- Overdue deals (from morning-brief.md if fresh <4hrs, else skip)
+
+## Step 4: Output (3 lines max)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)